From 2b7a4c79894442a468bf33d681cfc66133b4dbac Mon Sep 17 00:00:00 2001
From: Pankaj Dev <pankaj.dev@st.com>
Date: Wed, 23 Nov 2022 20:19:55 +0530
Subject: [PATCH] arm64: dts: st: add USB3DRD support in stm32mp257f-ev1

Add the required node(s) for USB3-DWC3 support : USB3DRD

Signed-off-by: Pankaj Dev <pankaj.dev@st.com>
Change-Id: I3c60586ce242e1afe65cf144deff1ecf0e984917
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index b07bd4ed8c6a..6dfe4ced4b19 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -114,3 +114,12 @@ &usb2_phy1 {
 &usb2_phy2 {
 	status = "okay";
 };
+
+&usb3dr {
+	status = "okay";
+
+	dwc3: usb@48300000 {
+		maximum-speed = "high-speed";
+		dr_mode = "peripheral";
+	};
+};
-- 
2.39.5

